notNeededPackages.json

```diff
@@ -3598,6 +3598,10 @@
             "libraryName": "react-tooltip",
             "asOfVersion": "4.2.4"
         },
+        "react-vega": {
+            "libraryName": "react-vega",
+            "asOfVersion": "7.0.0"
```

Comment:
According to the package.json, types were not included until v7.4.2: https://github.com/vega/react-vega/blob/v7.4.2/packages/react-vega/package.json#L14

Reply:
please check via `npm`, the tags in the repos are not always accurate: `npm pack react-vega@7.0.0`

Reply:
Oh, I see... I thought typescript required the "types" property, but it'll pick them up if the file is named correctly, too. My bad.